### PR TITLE
[AdminBundle][Improvment] Exclude certain stuff from the exception list

### PR DESCRIPTION
**Pull request by cv65kr**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #1762 

Example:
```yaml
kunstmaan_admin:
    admin_exception_excludes:
        - '/^\.|\.jpg$|\.gif$|.png$/i'
```

### CODE REVIEW

File: src/Kunstmaan/AdminBundle/EventListener/ExceptionSubscriber.php [hunk 1]
```diff
@@ -11,29 +11,60 @@
...
 
+    /**
+     * @param GetResponseForExceptionEvent $event
+     */
     public function onKernelException(GetResponseForExceptionEvent $event)
     {
         $request = $event->getRequest();
         $exception = $event->getException();
 
         if ( $exception instanceof HttpExceptionInterface ) {
```
Comment (sandergo90) [inline src/Kunstmaan/AdminBundle/EventListener/ExceptionSubscriber.php]: Can you remove the spaces here?

File: src/Kunstmaan/AdminBundle/EventListener/ExceptionSubscriber.php [hunk 2]
```diff
@@ -11,29 +11,60 @@
...
+                });
+
+                if(count($excludes) > 0) {
+                    return ;
+                }
+            }
 
             $hash = md5(
-                $exception->getStatusCode() . $request->getUri() . $request->headers->get('referer')
+                $exception->getStatusCode() . $uri . $request->headers->get('referer')
```
Comment (sandergo90) [inline src/Kunstmaan/AdminBundle/EventListener/ExceptionSubscriber.php]: No spaces between dots.

**Comment by cv65kr** [comment]
@sandergo90 done.

**Comment by sandergo90** [comment]
@cv65kr one last thing, can you add some docs to the upgrade guide ? Thanks!

**Comment by sandergo90** [comment]
@cv65kr and while you're on it, can you change the URL field to a text field instead of string?

**Comment by cv65kr** [comment]
@sandergo90, done also i change string to text for urlReferer field beacuse is similar.

**Pull request merged**